server/portal/apps/request_access/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+.. module:: portal.apps.site_search.urls
+   :synopsis: Site Search URLs
+"""
+from django.urls import re_path
+from portal.apps.request_access.views import IndexView
+
+app_name = 'request_access'
+urlpatterns = [
+    re_path('', IndexView.as_view(), name='index'),
```

Comment:
This could just be `path`, I think
```suggestion
    path('', IndexView.as_view(), name='index'),
```